### PR TITLE
Support Bootstrap icons

### PR DESCRIPTION
**Pull request by yiwen101**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [ ] Bug fix
- [x] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:
 

**Overview of changes:**
Support bootstrap icons #1977

**Anything you'd like to highlight/discuss:**
I need to upload the copy of bootstrap-icons to the expected folder to pass test (and in current code, the expected folder also contains the source code of all other icons supported) That is where the +50 000 comes from.

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**
Support Bootstrap icons

---

**Checklist:** :ballot_box_with_check:

- [x] Updated the documentation for feature additions and enhancements
- [x] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No unrelated changes 

---

**Reviewer checklist:**

Indicate the [SEMVER](https://semver.org/) impact of the PR:
- [ ] Major (when you make incompatible API changes)
- [ ] Minor (when you add functionality in a backward compatible manner)
- [ ] Patch (when you make backward compatible bug fixes)

At the end of the review, please label the PR with the appropriate label: `r.Major`, `r.Minor`, `r.Patch`.

Breaking change release note preparation (if applicable):
- To be included in the release note for any feature that is made obsolete/breaking

> Give a brief explanation note about:
>   - what was the old feature that was made obsolete
>   - any replacement feature (if any), and
>   - how the author should modify his website to migrate from the old feature to the replacement feature (if possible).

### CODE REVIEW

**Comment by damithc** [comment]
Thanks for attempting this @yiwen101 
This will be a good addition to the different icon sets we support.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
   "name": "root",
   "private": true,
-  "workspaces": ["packages/*"],
+  "workspaces": [
```
Comment (yucheng11122017) [inline package.json:4]: Unnecessary change

**Review by yucheng11122017 — COMMENTED** [review]
Thanks for the work @yiwen101! 
I think this looks good and the bootstrap icons are very cute. 

Small nit about documentation: Could you that bootstrap icons to the list [here ](https://deploy-preview-2494--markbind-master.netlify.app/userguide/formattingcontents#icons)and also add it to the acknowledgement

File: docs/userGuide/syntax/icons.md [hunk 2]
```diff
@@ -55,6 +55,12 @@ MarkBind currently supports Version 6 of Font Awesome (Free plan). For detailed
 1. You may also append `~class-name` to the end of the octicon name to add `class="class-name"` property to your Octicon (e.g. `:octicon-git-pull-request~icon-large-red:` will generate an Octicon of class *icon-large-red*). You may then add corresponding CSS to `{root}/_markbind/layouts/{layout-name}/styles.css` to customize the style of your Octicon.
 1. If your background is dark, you may use `:octiconlight-*:` to render the icon as white. 
 
+###### Using Bootstrap icons
```
Comment (kaixin-hc) [inline docs/userGuide/syntax/icons.md:58]: 1. Lines 3-5 : can you also update in this PR? Version of bootstrap relied on + Maybe delete line 5, which seems too repetitive. 
2. I never noticed before how repetitive these instructions are. I wonder if there is a better way to streamline the instructions? Hmm. Maybe a tooltip or something to see some different example icons would be cool.

**Review by kaixin-hc — APPROVED** [review]
lgtm, though see my comment and update the documentation further!

File: docs/userGuide/syntax/icons.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+<small>%%Acknowledgement: Font Awesome icons are provided by [Font Awesome](https://fontawesome.com/) under their [free license](https://fontawesome.com/license), Glyphicons are provided by [Glyphicons](https://glyphicons.com/) via [Bootstrap 3](https://getbootstrap.com/docs/3.3/). Bootstrap icons are designed by [@mdo](https://github.com/mdo), maintained by the [Bootstrap Team](https://github.com/orgs/twbs/people) and provided under MIT liscense. [Octicons](https://octicons.github.com) are copyright of GitHub, and Material icons are provided by [Google Fonts](https://fonts.google.com/icons) via [`material-icons` by Ravindra Marella](https://www.npmjs.com/package/material-icons) under the [Apache license 2.0](https://www.apache.org/licenses/LICENSE-2.0.html).%%</small>
 
 MarkBind supports using Font Icons provided by Font Awesome, Glyphicons and GitHub's Octicons.
```
Comment (yucheng11122017) [inline docs/userGuide/syntax/icons.md:5]:
```suggestion
MarkBind supports using Font Icons provided by Font Awesome, Glyphicons, GitHub's Octicons and Bootstrap icons.
```

**Review by yucheng11122017 — APPROVED** [review]
LGTM except small nit

**Comment by damithc** [comment]
Feature in action https://nus-cs2103-ay2324s2.github.io/website/admin/tp-w9.html

image
